util/settings/settings.go

```diff
@@ -273,6 +273,8 @@ const (
 	settingsWebhookGogsSecretKey = "webhook.gogs.secret"
 	// settingsApplicationInstanceLabelKey is the key to configure injected app instance label key
 	settingsApplicationInstanceLabelKey = "application.instanceLabelKey"
+	// settingsAppIdTrackingMethodKey is the key to configure tracking method for application resources
+	settingsAppIdTrackingMethodKey = "application.appIdTrackingMethod"
```

Comment:
I think `appId` is redundant since the setting is already prefixed with `application`. What do you think about `application.resourceTrackingMethod`? I think this name makes it self-explanatory that setting manages how application resources are being tracked.

@jannfis WDYT?

Reply:
agree,